### ISSUE TITLE
Add a Footer

### ISSUE THREAD

**Issue by foreshubham** [foreshubham/Signature-Size-Resizer#2]
We need a footer for the Signature Resizer project. The footer should include links to the project's GitHub repository, contact information, and social media links.

Requirements:
- The footer should be responsive and adapt to different screen sizes.
- It should include sections for navigation links, contact info, and social media icons.

Helpful Resources:
- [Material-UI Footer Example](https://material-ui.com/components/footer/)
- [Bootstrap Footer Documentation](https://getbootstrap.com/docs/5.1/examples/sticky-footer/)

**Comment by SufyaanKhateeb** [comment]
Hey @foreshubham,

I'm planning to work on this issue. Can you please assign me this issue?
Please feel free to provide any suggestions or guidance as I work on this task. It would be nice if you have and design for this, if not you can suggest what you want and I'll try to implement it. I'm open to feedback and would appreciate any insights you might have.

Thank you!

**Comment by foreshubham** [comment]
There is no specific design for this. But make sure we'll be developing or extending this website with more features so try to add some links as site mapping. The footer should contain social media icons and an email newsletter section. You can start working for this.

**Comment by SufyaanKhateeb** [comment]
Hello @foreshubham,
I went through the repository code and wanted to share some feedback. While I appreciate the effort put into the tool, I am having difficulty seeing the complete vision behind it. The primary function of this tool appears to be cropping images, a feature that most operating systems already provide through built-in tools.

Please understand that I am not trying to criticize your work. This application could indeed be useful for some people. However, I believe there might be more impactful problems to solve that could benefit from your skills and effort.

**Comment by foreshubham** [comment]
I agree with you. I can explain my vision behind this project before that,
May I know where are you from? It will help me explain things briefly.
